src/Helpers/Platform/PlatformHelper.js

```diff
@@ -11,11 +11,13 @@ export function getPlatform(platformId) {
   return api.showSource(platformId);
 }
 
-export function getPlatformItems(apiProps) {
+export function getPlatformItems(apiProps, options) {
   let apiPromise = null;
 
   if (apiProps) {
-    apiPromise = fetch(`${TOPOLOGICAL_INVENTORY_API_BASE}/sources/${apiProps}/service_offerings?archived_at=`).then(data => data.json());
+    apiPromise = fetch(`${TOPOLOGICAL_INVENTORY_API_BASE}/sources/${apiProps}/service_offerings?archived_at=${options
```

Comment:
Wanted to check on the reason why we are using the fetch and not the generated api client

Reply:
The client removes some attributes we need to get all relevant data (they are visible in the actual network response but the client removes them before it sends the response to the app). Until that is polished we need to sometimes use direct calls.